model-api-gen/src/main/kotlin/org/modelix/metamodel/generator/GeneratorInput.kt

```diff
@@ -220,6 +227,7 @@ internal class ProcessedConcept(
     var abstract: Boolean,
     val extends: MutableList<ProcessedConceptReference>,
     override var deprecationMessage: String?,
+    val metaInfo: Map<String, String>?,
```

Comment:
Is it necessary to make this nullable? Can't that always be an empty map if nothing provided? That would simplify usage for clients.